lib/tests/2wp.js

```diff
@@ -95,6 +95,58 @@ const execute = (description, getRskHost) => {
 
     });
 
+    it('should do a basic legacy pegin with the value above minimum', async () => {
+
+      // Arrange
+
+      const initialBridgeBalance = Number(await rskTxHelper.getBalance(BRIDGE_ADDRESS));
+      const initialBridgeUtxosBalance = await getBridgeUtxosBalance(rskTxHelper);
+      const initialFederationAddressBalanceInSatoshis = await getBtcAddressBalanceInSatoshis(btcTxHelper, federationAddress);
+      const senderRecipientInfo = await createSenderRecipientInfo(rskTxHelper, btcTxHelper);
+      const initialSenderAddressBalanceInSatoshis = await getBtcAddressBalanceInSatoshis(btcTxHelper, senderRecipientInfo.btcSenderAddressInfo.address);
+      // Value above minimum
+      const peginValueInSatoshis = minimumPeginValueInSatoshis + Number(btcToSatoshis(0.1));
+
+      // Act
+
+      const btcPeginTxHash = await sendPegin(rskTxHelper, btcTxHelper, senderRecipientInfo.btcSenderAddressInfo, satoshisToBtc(peginValueInSatoshis));
+      await ensurePeginIsRegistered(rskTxHelper, btcPeginTxHash);
+
+      // Assert
+
+      // The btc pegin tx is already marked as processed by the bridge
+      const isBtcTxHashAlreadyProcessed = await bridge.methods.isBtcTxHashAlreadyProcessed(btcPeginTxHash).call();
+      expect(isBtcTxHashAlreadyProcessed).to.be.true;
+
+      // The pegin_btc event is emitted with the expected values
+      const recipient1RskAddressChecksumed = rskTxHelper.getClient().utils.toChecksumAddress(ensure0x(senderRecipientInfo.rskRecipientRskAddressInfo.address));
+      const expectedEvent = createExpectedPeginBtcEvent(PEGIN_EVENTS.PEGIN_BTC, recipient1RskAddressChecksumed, btcPeginTxHash, peginValueInSatoshis);
+      const btcTxHashProcessedHeight = Number(await bridge.methods.getBtcTxHashProcessedHeight(btcPeginTxHash).call());
+      const peginBtcEvent = await findEventInBlock(rskTxHelper, expectedEvent.name, btcTxHashProcessedHeight);
+      expect(peginBtcEvent).to.be.deep.equal(expectedEvent);
```

Comment:
This could also go into it's own method, right?

Reply:
Updated.